hack/version.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+#
+# Copyright 2014 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#
+# (Loosely) used https://github.com/kubevirt/kubevirt/blob/master/hack/version.sh as a template
+
+# Grovels through git to return build version.
+source hack/common.sh
+
+function ovscni::version::get_version_vars() {
```

Comment:
i think that in bash you either define function with `function` or `()`, not both.

Reply:
Maybe though I copy pasted that from the original file. I can check if I can change it as you suggest. (I am not an expert in shell style so I can't make a good judgement if any of variants is better.)

Reply:
Me neither. Looks like we use both in other scripts under cluster/ too. Let's keep it.